quickstarts/microsoft.netapp/anf-sap-hana/anf-sap-hana-storage/azuredeploy.json

```diff
@@ -0,0 +1,642 @@
+            }
+        },
+        "AdditionalCapacityForSnapshots(% ofRAM)": {
+            "type": "int",
+            "minValue": 0,
+            "maxValue": 200,
+            "defaultValue": 50,
+            "metadata": {
+                "description": "Additional memory to store snapshots, must be specified as % of RAM (range 0-200). This is used to auto compute storage size."
+            }
+        },
+        "StartingSAPHostID": {
+            "type": "int",
+            "minValue": 1,
+            "maxValue": 99999,
+            "defaultValue": 1,
+            "metadata": {
+                "description": "Starting SAP HANA Host ID. Host ID 1 indicates Master Host. Shared, Data Backup and Log Backup volumes are only provisioned for Master Host i.e. HostID == 1."
+            }
+        },
+        "NumberOfSAPHosts": {
+            "type": "int",
+            "minValue": 1,
+            "defaultValue": 1,
+            "maxValue": 5,
+            "metadata": {
+                "description": "Total Number of SAP HANA Hosts in this deployment."
+            }
+        },
+        "SystemRole": {
+            "type": "string",
+            "allowedValues": [
+                "PRIMARY",
+                "HA"
+            ],
+            "defaultValue": "PRIMARY",
+            "metadata": {
+                "description": "The type of replication to use for the storage account. Primary indicates first of a SAP HANA System Replication (HSR) setup or No HSR. High Availability (HA) specifies local scenario."
+            }
+        },
+        "Prefix": {
+            "type": "string",
+            "minLength": 1,
+            "maxLength": 1024,
+            "defaultValue": "default",
+            "metadata": {
+                "description": "All volume names will be prefixed with the given text. The default values for prefix text depends on system role. For PRIMARY it will be \"\", HA it will be \"HA-\"."
+            }
+        },
+        "TagKey": {
+            "type": "string",
+            "defaultValue": "",
+            "maxLength": 512,
+            "metadata": {
+                "description": "If a Tag Key is specified, following TAG key will be added to each volume. It is recommended to add tag for HSR deployments, with tag name as HSRPartnerStorageResourceId."
+            }
+        },
+        "TagValue": {
+            "type": "string",
+            "defaultValue": "",
+            "maxLength": 256,
+            "metadata": {
+                "description": "If a Tag Value is specified, following TAG value will be added to each volume. The value will only be added if Tag Key was specified. It is recommended to add tag for HSR deployments, with tag value as \"Please enter the peering partner Volume ID\" and later updated for each volume."
+            }
+        },
+        "AzureNetAppFilesLocation": {
```

Comment:
Should include default value or parameterized.

Reply:
Done